### DISCUSSION TITLE
1984 개발일정 23-04 1week

### DISCUSSION THREAD

**Discussion by fkdl0048** [The-Day-In-1984/1984_Project#55] [General]
각 파트 별 개발일정을 아래 쓰레드에 적어주시기 바랍니다..!

![GithubCatGIF](https://user-images.githubusercontent.com/84510455/229838412-e1401c64-4871-4263-a104-c381d86d8c7d.gif)

**Comment by fkdl0048** [comment]
- https://github.com/The-Day-In-1984/1984_Project/issues/41

이번 주 개발일정이 없어서 뭘 할까 하다가 코드를 동작과 기능으로 좀 더 세분화해서 구현해볼려고 합니다.

**Comment by ccmuk58** [comment]
- #52

엑셀 파일을 게임에서 사용 가능한 형태로 변환하는 기능을 구현할 예정입니다.

 - #74

~~여유가 될 경우, 엑셀 파일 예제 작성 및 엑셀 파일 작성 가이드 문서 추가 예정입니다. (스토리 관련 데이터테이블)~~

**Comment by kyoonge** [comment]
- #62 

사다리 오르내리기 먼저 구현하고.... 잘되면 캐릭터 움직임 구조도 고쳐볼게요!

**Comment by dkssu46** [comment]
- #69 

미션 템플릿 작성 완료해놓겠습니다.
추가사항 생기면 수정 예정!

**Comment by aminseo1023** [comment]
- #72 
사임 캐릭터 설정 추가. 스토리보드(흐름도) 완성하겠습니다.
가능하다면 날짜별 스토리 구상도 완료해서 정리해놓을게요...!

**Comment by fkdl0048** [comment]
@aminseo1023 @dkssu46   

지금 방식이 잘 진행되고 있는 것 같아서 좋아!

혹시 가능하다면 다음 회의 전까지 게임의 전체적인 흐름을 파악할 수 있는 플로우 차트같은게 있으면 좋을 것 같아

지금은 컨텐츠 단위인데 전체적인 게임 흐름이 잡힌 문서가 있으면 좋을 것 같아요~!

**Reply by aminseo1023** [reply]
오케이 그럼 그것까지 해볼게요~!

**Reply by dkssu46** [reply]
알겠습니닷!

**Comment by kangjieum** [comment]
월요일 (4/10) 전까지 캐릭터 5명 2차 시안 올려놓겠습니다 !

**Comment by sueyeoun** [comment]
월요일 (4/10) 전까지 올려주신 레퍼런스들 천천히 다 듣고 전채적인 가상악기 서치 등 큰 틀 잡아보겠습니다 !
